jest.config.js

```diff
@@ -3,6 +3,9 @@ module.exports = {
     testMatch: [
         '**/tests/**/*.test.js',
     ],
+    modulePathIgnorePatterns: [
+        'testing/helpers/tests/.*test.js$'
```

Comment:
It will be better to move helpers and tests to utils/testcafe and this line will be not necessary.